### PR TITLE
feat: hit only once every 30mins the fs for release

### PR DESCRIPTION
**Pull request by Bilb**
once we've already fetched the latest release:

this but with 30minutes instead of 5:
Screenshot from 2024-07-26 11-00-25

### CODE REVIEW

File: ts/session/fetch_latest_release/index.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { isEmpty, isString } from 'lodash';
+import { ipcRenderer } from 'electron';
+import { DURATION } from '../constants';
+import { getLatestReleaseFromFileServer } from '../apis/file_server_api/FileServerApi';
+
+/**
+ * Try to fetch the latest release from the fileserver every 1 minute.
+ * If we did fetch a release already in the last 30 minutes, we will skip the call.
+ */
+const fetchReleaseFromFileServerInterval = DURATION.MINUTES * 1;
```
Comment (yougotwill) [inline ts/session/fetch_latest_release/index.ts:14]: not 5 minutes?

Comment (Bilb) [reply ts/session/fetch_latest_release/index.ts:14]: nah, better to have it happen fast and then fall back to the 30minutes

Comment (yougotwill) [reply ts/session/fetch_latest_release/index.ts:14]: Cool

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**